### DISCUSSION TITLE
Sysmon Ingest Pipeline Issue with ProcessId

### DISCUSSION THREAD

**Discussion by El-Reno** [Security-Onion-Solutions/securityonion#1918] [General] [answered]
I've noticed that Sysmon event logs are not properly renaming Process IDs from winlog.event_data.ProcessId. Looking at the Elasticsearch ingest file for Sysmon, the ingest does not correctly identify the true Sysmon field. The ingest pipeline is looking for ProcessID and processID. The correct Sysmon field should be ProcessId. Adding the following line after the last ProcessID line in sysmon ingest corrects the issue:
`{ "rename":            { "field": "winlog.event_data.ProcessId",                            "target_field": "process.pid",          "ignore_missing": true  } }`

**Comment by weslambert** [comment]
We'll take a look at this - thanks!

**Comment by defensivedepth** [comment, marked as answer]
Just merged in this fix, thanks for pointing this out!

https://github.com/Security-Onion-Solutions/securityonion/pull/1954

**Reply by El-Reno** [reply]
No problem! Thank you guys for making an awesome product!
